client/main/interfaces/IInputOutput.ts

```diff
@@ -0,0 +1,2 @@
+export type IInputFilter = Record<string, any>;
```

Comment:
Is this "any" something we should add typing to later? Or is it meant to just be literally anything?

Reply:
Yeah, we will add typing to this later.